.github/workflows/01-php-lint.yml

```diff
@@ -5,6 +5,8 @@ on:
       branches:
           - trunk
     pull_request:
+      paths:
+        - '**.php'
```

Comment:
for our gitlab pipeline we use the following changed paths:
```
              - composer.json
              - .bc-exclude.php
              - 'changelog/**/*.md'
              - 'src/**/snippet/**/*.json'
              - '**/*.php'
```
Would be great if we can use the same rules here

https://github.com/shopware/platform/blob/4011bd2b446c0e5d8bdf05fd327ffea011d928fe/.gitlab/stages/10-lint.yml#L154

Reply:
Yes that sounds reasonable, the only thing I am not sure about, is it on purpose that linting should not be run on the php files in the root directory? Would it not be better to use something like:
```
        - composer.json
        - 'changelog/**/*.md'
        - 'src/**/snippet/**/*.json'
        - '**.php'
```

Reply:
Good catch :+1: can you adjust that as well for the gitlab pipeline?

Reply:
Done :-)